src/com/sun/ts/tests/compat12/ejb/Tx_Single/TestBeanEJB.java

```diff
@@ -26,7 +26,7 @@
 import java.util.*;
 import java.rmi.*;
 import jakarta.ejb.*;
-import javax.transaction.*;
+import jakarta.transaction.*;
 
```

Comment:
weird that I missed some in my pr, so this one is needed.

Reply:
This project is monster. Think that noone expect that PRs at this phase will cover everything. I just wanted to reduce build issues.